instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/builder/internal/DefaultHttpServerInstrumenterBuilder.java

```diff
@@ -53,7 +54,7 @@ public final class DefaultHttpServerInstrumenterBuilder<REQUEST, RESPONSE> {
       httpAttributesExtractorBuilder;
   private final HttpSpanNameExtractorBuilder<REQUEST> httpSpanNameExtractorBuilder;
 
-  @Nullable private final TextMapGetter<REQUEST> headerGetter;
+  @Nullable private TextMapGetter<REQUEST> headerGetter;
```

Comment:
it looks like this can stay `final`?

Reply:
yes - added factory method to make it possible